sr_object_segmentation/scripts/segmentation.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import rospy
+
+from sr_object_segmentation.hsv_segmentation import HSVSegmentation
+from sr_object_tracking.utils import Utils
+
+from sensor_msgs.msg import Image, RegionOfInterest
+
+
+class Segmentation(object):
+    def __init__(self):
+        rospy.init_node('sr_object_segmentation')
+        self.color = rospy.get_param('/color')
+
+        self.image_sub = rospy.Subscriber("/camera/rgb/image_color", Image, self.spin)
+        self.selection_pub = rospy.Publisher("/roi/segmented_box", RegionOfInterest, queue_size=1)
+
+        self.seg = HSVSegmentation(self.color)
+        self.utils = Utils()
+
+        self.frame = None
+
+    def spin(self, data):
+        """
+        Convert the ROS image to OpenCV format using a cv_bridge helper function and make a copy
+        """
+        self.frame = self.utils.convert_image(data, "bgr8")
+        stop_seg = rospy.get_param('/stop_seg')
```

Comment:
this is a bad way of using ros parameters. In ROS parameters are static (as in that's what you read when starting the node). You should use a service here